pkg/engine/jmespath/functions.go

```diff
@@ -54,6 +56,24 @@ const genericError = errorPrefix + "%s"
 const zeroDivisionError = errorPrefix + "Zero divisor passed"
 const nonIntModuloError = errorPrefix + "Non-integer argument(s) passed for modulo"
 
+var layouts = map[string]string{
```

Comment:
Do you mean we should use the string values of the constants instead?

Reply:
Yes - both of these are string values, just one is taken from the `time` package. So using the user input seems better to me.

Reply:
I am still not super happy with this - WDYT if we just let users set the `layout` themselves?

So we basically offer users to make the exact inputs as `time.Parse` takes. I think it would make the implementation cleaner & offer users a known interface.

Reply:
We could do that or we could keep some of the standard ones like the RFC822 and loose the others. Maybe @JimBugwadia could weigh in.

Reply:
Sure, we've removed the map.